libc-test/build.rs

```diff
@@ -534,6 +539,16 @@ fn main() {
             // On Mac we don't use the default `close()`, instead using their $NOCANCEL variants.
             "close" if apple => true,
 
+            // Definition of those functions as changed since unified headers from NDK r14b
```

Comment:
Yeah, and when fixing that issue, we'll certainly reword that comment in any way :)